### ISSUE TITLE
Cannot read property isLoaded.

### ISSUE THREAD

**Issue by pollyolly** [shripalsoni04/nativescript-webview-interface#55]
Platform: Nativescript Vue
Version: 6.2.0
Plugin: nativescript-webview-interface (1.4.3)

Problem: Cannot read property isLoaded.

File: nativescript-webview-interface/index.android.js (line:50)

WebViewInterface.prototype._initWebView = function(src){
        var _this = this;
        **if(this.webView.isLoaded) {** 
            _this._setAndroidWebViewSettings(src);
        } else {
            var handlerRef = _this.webView.on('loaded', function(){
                _this._setAndroidWebViewSettings(src);
                _this.webView.off('loaded', handlerRef);
            });
        }
    };

**Comment by pollyolly** [comment]
I think this plugin is no longer supported the latest Nativescript and Android Runtime. 
Currently Im using it to this version.
Android Runtime: 6.2.0
Nativescript Cli: 6.0.3
Deployed in: Android 9

**Comment by ghost** [comment]
I use it on Android 10 | {N} 7, and it works like a charm , 

just change the plug-in ( file: index.android.js ) a bit:

line 2 from `const platformModule = require("platform");` to `const platformModule = require("@nativescript/core");`

and

line 78 from `if (platformModule.device.sdkVersion >= 19) {` to `if (platformModule.Device.sdkVersion >= 19) {`

**Comment by shripalsoni04** [comment]
Should have been fixed in the latest version @1.4.4

**Comment by artech53** [comment]
But I get this error now...
"@nativescript/core": "^7.0.3"
"nativescript-webview-interface": "^1.4.4"
Android API 28 Emulator AndroidStudio

nativescript-vue
I use that code:

```
<WebView ref="web"></WebView>
...
loaded() {
        const webView = this.$refs.web;
        const oLangWebViewInterface = new WebViewInterface(webView, URL);
        webView.on('anyEvent', (selectedLanguage) => {
            console.log('asd' + JSON.stringify(selectedLanguage));
        });

        console.log('LOAD END')
    }
```
